.github/workflows/package.yml

```diff
@@ -237,6 +239,7 @@ jobs:
     env:
       APACHE_ARROW_REPOSITORY: ${{ github.workspace }}/apache-arrow
     runs-on: ubuntu-latest
+    timeout-minutes: ${{ matrix.timeout-minutes || 60 }}
```

Comment:
I think 60 minutes is too long.
Most seem to be over within 20 minutes.

Reply:
fix: [d187b49](https://github.com/groonga/groonga/pull/1886/commits/d187b49dba3faec4f25556867a79ec012266154c) Thanks, I set the 30 mins as a default time using the following information.
re: https://github.com/groonga/groonga/actions/runs/10499754040/usage
ref: https://github.com/groonga/groonga/actions/runs/10507209401/usage